nsd/tclloop.c

```diff
@@ -36,6 +36,10 @@
 
 static const char *RCSID = "@(#) $Header: /Users/dossy/Desktop/cvs/aolserver/nsd/tclloop.c,v 1.4 2009/12/24 19:50:08 dvrsn Exp $, compiled: " __DATE__ " " __TIME__;
 
+#ifdef __APPLE__
+#include <ctype.h>
+#endif
```

Comment:
What is the reason for adding this? Please correct me if I’m wrong, but I don’t see usage of ctype.h functions/macros in tclloop.c. And ctype.h should already be included on macOS by nsd.h.